packages/client/src/v2-events/hooks/usePrintableCertificate.ts

```diff
@@ -37,32 +41,32 @@ import { getUserDetails } from '@client/profile/profileSelectors'
 import { getOfflineData } from '@client/offline/selectors'
 import { useEventConfiguration } from '../features/events/useEventConfiguration'
 import { useEvents } from '../features/events/useEvents/useEvents'
+import { useDrafts } from '../features/drafts/useDrafts'
+import { getEventDrafts } from '../features/events/components/Action/utils'
+import { hasStringFilename } from '../utils'
 
 async function replaceMinioUrlWithBase64(
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  declaration: Record<string, any>,
+  declaration: EventState,
   config: EventConfig
 ) {
+  // Clone to avoid mutating the original declaration
+  const declarationClone: EventState = cloneDeep(declaration)
```

Comment:
I'm quite sure that `: EventState` is unnecessary here, as cloneDeep retains the inferred type